jax/experimental/pallas/__init__.py

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Module for pallas, a JAX extension for custom kernels."""
+"""Module for Pallas, a JAX extension for custom kernels.
+
+See the Pallas documentation at https://jax.readthedocs.io/en/latest/pallas.html.
```

Comment:
Can we use a reST reference to link to the Pallas docs instead?

Reply:
I was thinking that actual links are helpful for those who read the sources.